lib/commands/deploynodeapp.js

```diff
@@ -495,14 +804,16 @@ function handleUploadResult(err, req, fileName, itemDone) {
 
 // Create a target endpoint that references the Node.js script
 function createTarget(opts, request, done) {
-  var targetDoc = mustache.render(
```

Comment:
Yes it was intentional. I moved away from using this `createTarget` method entirely.

Instead, when `edge-services` is targeted, it builds the proxy bundle all at once and uploads it, rather than doing 1 file at a time like the `trireme` impl does here.